docs/source/json-api/index.rst

```diff
@@ -1021,6 +1021,13 @@ Contracts Query Stream
 List currently active contracts that match a given query, with
 continuous updates.
 
+An optional ``offset`` returned by a prior query (see output examples
+below) may be specified *first*, as a separate body.  It must be a
+string, and if specified, the stream will begin immediately *after* the
+response body that included that offset::
+
+    {"offset": "5609"}
```

Comment:
I would move that block after the description of the "main" query format and replace *first* by something *before* the "main" query. My rationale is that I want the simple use case to come first and the more advanced stuff later. Other than that, the docs and the API are fine. Thanks a lot.

Reply:
Moved to just before we start talking about output in e6995900820f8938086428468eb8627902072729